src/index.ts

```diff
@@ -7,20 +7,24 @@ import {initialize} from './lib/initialize';
 import {load} from './lib/load';
 import {parameterize} from './lib/parameterize';
 
+import {debugLogger} from './util/debug-logger';
 import {parseIEProcessArgs} from './util/args';
 import {andHandle} from './util/helpers';
 import {logger} from './util/logger';
 import {validateManifest} from './util/validations';
 
 import {STRINGS} from './config';
 
-const {DISCLAIMER_MESSAGE} = STRINGS;
+const {DISCLAIMER_MESSAGE, STARTING_IMPACT_FRAMEWORK, EXITING_IF} = STRINGS;
```

Comment:
is exiting if, then starting if

Reply:
@manushak you got this wrong. in starting you have written impact framework, however exiting if. So for fixing this inconsistency please use if in both places.

Reply:
Uh, I've got it now. The variable names are consistent with the message itself. 
@jmcook1186, maybe we need to rename 'Starting Impact framework' to 'Starting IF' to be consistent with 'Exiting IF'. What do you think?

Reply:
@narekhovhannisyan  renamed